src/extract/src/html/extract-regexps-html.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+# Description: This file takes in a html file from extract-regexes.pl, finds all the script 
+# tags, and combine the JS in them into a temporary js file. It then sends the path of the
+# temporary js file back to extract-regexes.pl to let it pipeline the js file to the javascript
+# extractor. After extract-regexes.pl finishes extracting, The temporary JS file will be 
+# deleted by extract-regexes.pl. 
+
+from bs4 import BeautifulSoup
+import sys
+import subprocess
+import json
+
+file_path = sys.argv[1]
+with open(file_path) as fp:
+    soup = BeautifulSoup(fp, 'html.parser')
+
+js_from_html = ''
+for script in soup.find_all('script'):
+    js_from_html += script.string
+
+# create temp-js-content.js based on the location of extract-regexes.pl
```

Comment:
This will not work well if multiple instances of vuln-regex-detector are invoked concurrently. Use the Python tempfile library to make a temp dir.